src/main/java/core/basesyntax/dao/impl/MessageDaoImpl.java

```diff
@@ -3,30 +3,100 @@
 import core.basesyntax.dao.MessageDao;
 import core.basesyntax.model.Message;
 import java.util.List;
+import javax.persistence.criteria.CriteriaQuery;
+import lombok.extern.log4j.Log4j;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
 
+@Log4j
 public class MessageDaoImpl extends AbstractDao implements MessageDao {
     public MessageDaoImpl(SessionFactory sessionFactory) {
         super(sessionFactory);
     }
 
     @Override
-    public Message create(Message entity) {
-        return null;
+    public Message create(Message message) {
+        log.info("Calling a create() method of MessageDaoImpl class");
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(message);
+            log.info("Attempt to store message " + message + " in db.");
+            transaction.commit();
+            return message;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't create message entity. ", e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
     }
 
     @Override
     public Message get(Long id) {
-        return null;
+        log.info("Calling a get() method of MessageDaoImpl class");
+        Transaction transaction = null;
```

Comment:
I think you don't need to create a null Transaction if you don't use it in this method.